pkg/build/iso_test.go

```diff
@@ -0,0 +1,33 @@
+package build
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/suse-edge/edge-image-builder/pkg/config"
+)
+
+func TestGenerateXorrisoArgs(t *testing.T) {
```

Comment:
Don't scrutinize this yet, it's going to be converted into a test for running the xorriso command and making sure that was given the right arguments.